person.rb

```diff
@@ -13,9 +15,43 @@ def of_age?
     (@age.is_a? Numeric) && (@age >= 18)
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def can_use_services?
     @age >= 18 || @parent_permission
   end
 end
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- Hey :wave: , let's also add the getter/setter method for the nameable instance variable so one could able to access it. `attr_accessor :nameable`

Reply:
Sure I will add the getter and setter for `nameable` using `attr_accessor`